### ISSUE TITLE
Add Slides URL filler to Pronto

### ISSUE THREAD

**Issue by asgrim** [rdohms/pronto#26]
If tightenco/symposium#122 gets implemented, it would naturally be great if Pronto could autofill slides URLs :)

**Comment by elazar** [comment]
tightenco/symposium#122 has been implemented and is now in production. Progress on this issue may begin now.

**Comment by elazar** [comment]
Could be wrong, but I think this is the only change that's needed. The new `slides` field is documented in the [related API endpoint documentation](https://github.com/tightenco/symposium/blob/master/doc/api.md#response-3). "Slides" is the field label in OpenCFP, so this change should cover that use case.

I'm not overly familiar with how to build Chrome extensions, so I don't know how to test this myself. The project README could stand some instructions in that regard. @rdohms can you confirm this change will work as intended?

``` diff
diff --git a/src/js/form/alternate_names.js b/src/js/form/alternate_names.js
index 2021d21..10982df 100644
--- a/src/js/form/alternate_names.js
+++ b/src/js/form/alternate_names.js
@@ -5,6 +5,7 @@ export default {
         description: ['abstract', 'describe', 'summary'],
         organizer_notes: ['notes', 'additional', 'why accept', 'comments'],
         level: ['audience'],
+        slides: ['slides', 'slides url'],
         type: ['talk type'],
         body: ['bio', 'about you', 'biography']
     },
```

**Comment by rdohms** [comment]
@elazar yes that should be the only change needed.
I'm working on a big rewrite and that will hopefully make the codebase better to deal with and allow more people to work on it, hence the lack of README content. I was pretty much shooting in the dark with chrome extensions and js myself.

You want to open a PR with that?

**Comment by elazar** [comment]
@rdohms Done. :smile:

**Comment by rdohms** [comment]
:+1:

**Comment by asgrim** [comment]
Awesome! Thanks @rdohms and @elazar

**Comment by rdohms** [comment]
@elazar I just ran some test queries against the API here and "slides" is not coming in in the `api/user/<id>/talks`endpoint.

I just added slide info to my talk so it may be some cache issue between saving/api output, but its not a pronto cache or anything, i tried using postman to check.

Can you double check that this was added to Symposium API?

**Comment by mattstauffer** [comment]
@rdohms was added, possible it's not being output for some strange reason, will look in a bit

**Comment by rdohms** [comment]
Fixed in release v0.4.3
